### PR TITLE
Fixed redirect to home by clicking on the logo, fixed 404 page button redirect to home page

### PR DESCRIPTION
**Pull request by Senor-Ducky**
Fixed redirect to home/login page by clicking on the GITNOTER logo, fixed the button on 404 page before it wasn't redirecting to the intended page but now it does.

### CODE REVIEW

File: frontend/src/components/404.tsx [hunk 1]
```diff
@@ -39,7 +40,7 @@ const ErrorPage: React.FC = (): React.ReactElement => {
         md={7}
         lg={7}
         sx={{ display: "grid", placeContent: "center" }}
-        onClick={handleClick}
+        // onClick={handleClick}
```
Comment (vivekweb2013) [inline frontend/src/components/404.tsx:43]: Please delete the line instead of commenting

File: frontend/src/components/AppBar.tsx [hunk 2]
```diff
@@ -21,6 +21,9 @@ interface Props {
 
 
 
+
```
Comment (vivekweb2013) [inline frontend/src/components/AppBar.tsx:24]: Delete unnecessary line breaks

**Comment by Senor-Ducky** [comment]
Hey, it's done, I thought I had format on save enabled

**Comment by vivekweb2013** [comment]
@Senor-Ducky Thanks so much for addressing this issue. Could you please amend the commit message from "Fixed redirect to home by clicking on the logo, fixed 404 page button redirect to home page" to just "Fix nav link & 404 redirect button"

**Why?**
Because a good commit message should not exceed 50 characters
Also a commit message should use the imperative, present tense. So use **Fix** instead of **fixed**

**How to amend the commit message?**
Just run the below commands
```
git commit --amend -m "Fix nav link & 404 redirect button"
git push --force
```

Let me know if you face any issues

**Comment by vivekweb2013** [comment]
oh you have multiple commits. maybe try this
```
git reset --soft HEAD~3
git commit -m "Fix nav link & 404 redirect button"
git push --force
```

**Comment by vivekweb2013** [comment]
@Senor-Ducky Excellent. thanks for the quick response. Let me merge the PR

**Pull request merged**